README.md

```diff
@@ -125,6 +125,155 @@ record = client.write(record_type, data)
 print 'Wrote record {0}'.format(record.meta.record_id)
 ```
 
+## Searching records
+
+E3DB supports complex search options for finding records based on the fields stored in record metadata. 
+
+For example, to list all records of type `contact` and print a simple report containing names and phone numbers:
+
+```python
+# setup
+import e3db
+from e3db import Search
+client = e3db.Client(' config ')
+
+query = Search(include_data=True).Match(record_type=['contact'])
+results = client.Search(query)
+
+for record in results:
+    full_name = "{0} --- {1}".format(record.data['first_name'], record.data['last_name'])
+    print "{0} --- {1}".format(full_name, record.data['phone'])
+```
+
+The full list of parameters you can search for are under `e3db.types.Params`. Searching gives you access to chaining more powerful queries such as conditional operators, exlusions, and date filters.     
+
+To search for records of type `season` that have values `summer` and `sunny`, create the following query:
+```python 
+# e3db setup...
+query = Search().Match(condition="AND", record_type=["season"], values=["summer", "sunny"])
+results = client.Search(query)
+```
+
+To search for records of type `soda` excluding values of `pepsi`, create the following query:
+```python 
+# e3db setup...
+query = Search().Match(record_type=["soda"]).Exclude(values=["pepsi"])
+results = client.Search(query)
+```
+
+To filter queries provide a datetime range and a valid timezone, to override the `UTC` default. To get all the records of last 24 hours written by a user use:
+```python
+# e3db setup...
+from datetime import datetime, timedelta
+
+# time T between after <-T-> before
+before = datetime.now()
+after =  before - timedelta(days=1)
+writer_id = "some_writer_uuid"
+
+query = Search().Match(writer=[writer_id]).Range(zone_offset="-08:00", after=after, before=before)
+results = client.Search(query)
+```
+
+#### Defaults
+
+The Search has a number of default parameters when searching, more detail can be found within the inline documentation.
+
+Under Search there are these defaults:
+```python
+# Starts search from the first page of results
```

Comment:
Should include documentation that this is an optional field, only needed for paginated queries.